scripts/update_metadata.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+'''
+@author: Patrick Cleeve
+
+This is a script to update the MD_POS_COR for the stage-bare component while odemis is running.
+
+run as:
+python3 ./scripts/update_metadata.py 
+
+and follow the instructions in the terminal.
+'''
+
+
+from odemis import model
+
+def main():
+
+    print(f"Calibration metadata updater")
```

Comment:
-> "METEOR calibration updater"?

"metedata" is really an implementation detail because in Odemis we started to call this sort of data "metadata", but that doesn't match the common English term of "metadata".